src/Modal.html

```diff
@@ -1,251 +1,245 @@
-<:Window on:keyup="onKeyup(event)"/>
+<svelte:window on:keyup="onKeyup(event)" />
 <div
   ref:modal
   class="svelte-modal"
   tabindex="-1"
-  data-center="{{ center }}"
-  data-hidden="{{ hidden }}"
-  style="z-index: {{ zIndex }}; opacity: {{ modalStyle.opacity }};"
->
+  data-center={center}
+  data-hidden={hidden}
+  style="z-index: {zIndex}; opacity: {modalStyle.opacity};">
   <div
     ref:content
     class="content"
-    style="transform: scale({{ contentStyle.scale }});"
-  >
-    <slot></slot>
-  </div>
-
-  <div on:click="onScrimPress()">
-    <slot name="scrim">
-      <Scrim/>
```

Comment:
The scrim should not be removed. It also needs to be updated to support Svelte 2.